doc/man3/EVP_PKEY_check.pod

```diff
@@ -21,6 +22,12 @@ EVP_PKEY_private_check, EVP_PKEY_pairwise_check
 EVP_PKEY_param_check() validates the parameters component of the key
 given by B<ctx>.
 
+EVP_PKEY_param_check_quick() also validates the parameters component of the key
```

Comment:
I think it needs to be understood in terms of how it is different to EVP_PKEY_param_check().  I've removed "also", but also reworded this a little. Hopefully it can now be read in isolation, but without losing the connection with EVP_PKEY_param_check().